fixtures/ghc-options-flow.cabal

```diff
@@ -0,0 +1,43 @@
+cabal-version:      3.4
+name:               simple-pkg
+version:            0.1.0.0
+
+common warnings
+    ghc-options:
+        -Wall
+        -Wtabs
+        -Wincomplete-uni-patterns
+        -Werror=dodgy-imports
+        -Werror=missing-home-modules
+        -Werror=star-is-type
+        -Werror=unused-imports
+        -eventlog
+        -O0
+        +RTS -A32M -RTS
```

Comment:
If you know of a way to keep `+RTS -A32M -RTS` as one line while options prefixed with `-W` as one per line then that could be very nice.